roles/pulp_common/defaults/main.yml

```diff
@@ -21,6 +21,7 @@ pulp_pip_editable: yes
 pulp_use_system_wide_pkgs: false
 prereq_pip_packages:
   - Jinja2
+  - psycopg2-binary
```

Comment:
Why are we doing this?

We switched a while ago.

b5fc031e44dd7e173d384737b1c30e21e4d2f3be

https://github.com/pulp/pulpcore/pull/378

We later did other changes related to psycopg2 as well, see `git log` and search it for "psycopg2".

Reply:
I forgot to remove it,
I was getting psycopg errors and I tried it, but the issue was the SCL and symlinking was the fix.
I'll remove it